sites/main-site/src/content/special-interest-groups/immunology/index.md

```diff
@@ -18,12 +18,17 @@ pipelines:
 
 # Introduction
 
-TBD
+The nf-core SIG Immunology aims to connect researchers and developers working on immunology-related pipelines within the nf-core community. While several pipelines relevant to immunology already exist, the community around them remains fragmented. This SIG seeks to bring together users and contributors to foster collaboration and knowledge exchange.
```

Comment:
```suggestion
The nf-core Special Interest Group Immunology aims to connect researchers and developers working on immunology-related pipelines within the nf-core community. While several pipelines relevant to immunology already exist, the community around them remains fragmented. This SIG seeks to bring together users and contributors to foster collaboration and knowledge exchange.
```
please use the full name instead of the initialism